services/src/main/java/org/jboss/windup/web/services/RuleDataLoader.java

```diff
@@ -285,6 +412,79 @@ private void loadRules(RulesPath rulesPath, Path path)
         }
     }
 
+    private void loadLabels(LabelsPath labelsPath, Path path)
+    {
+        boolean fileRulesOnly = labelsPath.getLabelsPathType() == PathType.USER_PROVIDED;
+
+        RuleLoaderContext ruleLoaderContext = new RuleLoaderContext(Collections.singleton(path), null);
+
+        LabelProviderRegistry labelProviderRegistry;
+        try
+        {
+            labelProviderRegistry = labelProviderService.loadLabelProviderRegistry(Collections.singleton(path), fileRulesOnly);
+        }
+        catch (Exception e)
+        {
+            LabelProviderEntity labelProviderEntity = new LabelProviderEntity();
+            labelProviderEntity.setProviderID("ERROR");
+            labelProviderEntity.setDateLoaded(new GregorianCalendar());
+            labelProviderEntity.setOrigin(path.toString());
+            labelProviderEntity.setLabelsPath(labelsPath);
+            labelProviderEntity.setLoadError(e.getMessage());
+            entityManager.persist(labelProviderEntity);
+            return;
+        }
+        LOG.info("Providers for: " + path + " are " + labelProviderRegistry.getProviders());
```

Comment:
to avoid any misunderstanding with the log from the rules loader
```suggestion
        LOG.info("Label providers for: " + path + " are " + labelProviderRegistry.getProviders());
```